contracts/gov/ERC721/ERC721Power.sol

```diff
@@ -162,6 +164,9 @@ contract ERC721Power is IERC721Power, ERC721Enumerable, Ownable {
 
         (newPower, collateral) = getNftPower(tokenId);
 
+        totalPower -= nftInfos[tokenId].currentPower;
```

Comment:
This is fine, but the `totalPower` should be updated including the `maxPower`. Otherwise the voting might be compromised

Reply:
done